### ISSUE TITLE
Add timestamp operation

### ISSUE THREAD

**Issue by chilland** [qntfy/kazaam#5]
It would be great if a timestamp field could be specified and a transformation applied to the string. For example you could transform a Unix timestamp to standard GMT. Probably could use the built-in golang date formatting as a template.

**Comment by ryanleary** [comment]
Have any proposals for what the json specification should look like?

**Comment by chilland** [comment]
Probably can just rely on the golang parsing for now. So something along the lines of:

```
{
  "operation": "timestamp",
  "format": "2006-01-02T15:04:05Z07:00",
  "spec": {
    "new_timestamp: "path.to.timestamp"
  }
}
```

**Comment by chilland** [comment]
closed by #44
